_posts/2021-02-02-sc20-report.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: RSE-HPC-2020 Workshop Report
+tags: [conference]
+posted_by: Charles Ferenbaugh, Daniel S. Katz, Sandra Gesing, and Simon Hettrick
+---
+
+We were excited to hold the inaugural [Research Software Engineers in HPC
+Workshop (RSE-HPC-2020)](https://us-rse.org/rse-hpc-2020/) as part of the
+[SC20](https://sc20.supercomputing.org/) conference this year! The half-day,
+virtual event featured 18 speakers and over 100 participants.
+
+The opening session, chaired by Charles Ferenbaugh, began with keynote addresses
+from Simon Hettrick in the UK and Frank Löffler in Germany. They spoke on the
+growth of the RSE movements in their respective countries. Though the name
+"Research Software Engineer" is only about eight years old, the role is much
+older. In those countries (and many others), when RSE events were first
+organized, many RSEs joined in right away and communities grew quickly. Many who
+joined said they were excited to have "found their tribe."
+
+Following the keynotes, three speakers gave lightning talks on their RSE work in
+various contexts: Miranda Mundt on how research software engineers can support
+scientific software, Emilio Carcamo on the Research Software Engineering
+ecosystem for scientific computing supported by SC3UIS in Colombia, and
+Bernadette Boscoe on RSEs as maintainers and their role in keeping code alive.
```

Comment:
Are these talks available anywhere (slides or video?)

Reply:
Some of the talks have slides posted on the workshop website.  I believe the videos are only available to people who registered for the conference.

Reply:
Link to slides:  https://us-rse.org/rse-hpc-2020/agenda/

Reply:
Should we add that link after mentioning the talks?

Reply:
Good idea!  Will do.